src/System.CodeDom/src/Microsoft/VisualBasic/VBCodeGenerator.PlatformNotSupported.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.CodeDom;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.VisualBasic
+{
+    partial class VBCodeGenerator
+    {
```

Comment:
Yes, so that you don't have to go searching for what other partial versions of this may be compiled in in order to determine what visibility is being applied.